.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "📖 Documentation"
+about: Add new documentation, or report missing, incorrect or unclear documentation
+title: ''
+labels: "documentation, awaiting triage"
+assignees: ''
+
+---
+
+<!--
+    Please fill in as much of the template below as you’re able to. If you're unsure whether the issue already exists or how to fill in the template, open an issue anyway. Our team will help you to complete the rest.
+
+    Your issue might already exist. If so, add a comment to the existing issue instead of creating a new one. You can find existing issues here: https://github.com/alphagov/govuk-frontend-docs/issues
+
+    We also welcome pull requests from users, if you feel comfortable doing so.
+-->
+
+## Related documentation
+<!-- Does this issue refer to a gap or mistake in some existing documentation? Provide a link if possible. -->
```

Comment:
By "gap," do we mean some entirely new content (like a full page) the user wants us to create, or a gap in existing docs?

If the former, then should we tweak this text to also cover requests for new content?

First go at alternative wording:

"Does this issue refer to existing documentation? If so, please provide a link to the documentation. Alternatively, if this issue requests new content, tell us what you'd like to see."

Reply:
I think it could refer to both - a missing section; an edge case that perhaps isn't covered or a piece of functionality that isn't documented at all.

The description of this template says "Add new documentation, or report missing, incorrect or unclear documentation" so it should cover both scenarios.